Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -327,9 +329,10 @@ void MolDraw2D::get2DCoordsMol(RWMol &mol, double &offset, double spacing,
   }
 
   const bool canonOrient = true;
+  const bool kekulize = false;
   RDDepict::compute2DCoords(mol, nullptr, canonOrient);
   MolDraw2DUtils::prepareMolForDrawing(
-      mol, false);  // don't kekulize, we just did that
+      mol, kekulize);  // don't kekulize, we just did that
```

Comment:
the comment was there for the naked bool. If you're going to name it, then please move the comment up to the definition of `kekulize`